File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    int row;
    int column;
    int space;
//check the input
    do
    {
        height = get_int("How tall is the pyramid? ");
    }
    while (height < 1 || height > 8);
//counts the rows
    for (row = 0; row < height; row++)
    {
        //prints the spaces
        for (space = 0; space < height - row - 1; space++)
        {
            printf(" ");
        }
        //prints the #
        for (column = 0; column <= row; column++)
        {
            printf("#");
        }
        printf("  ");

        for (column = 0; column <= row; column++)
        {
            printf("#");
        }
        printf("\n");

    }
}

File: sentimental-mario-more/mario.py
def main():
    height = get_height()

    for i in range(height):
        print(" " * (height - 1 - i), end="")
        print("#" * (i+1), end="")
        print(" " * 2, end="")
        print("#" * (i+1), end="\n")


def get_height():
    while True:
        try:
            n = int(input("How tall is the pyramid? "))
            if n > 0 and n < 9:
                break
            else:
                print('Use between 1-9')
        except ValueError:
            print('Use a NUMBER between 1-9')
    return n

main()